scripts/lib/setup/installers.sh

```diff
@@ -170,9 +180,25 @@ function install_android_sdk() {
 }
 
 function install_android_sdk_linux() {
-  cecho "@b@yellow[[+ Skipping Android SDK setup, not implemented on Linux]]"
-
-  return 1
+  if [ -e ~/Android ]; then
+    cecho "@b@yellow[[+ Skipping Android SDK setup- the ~/Android already exists ]]"
```

Comment:
can you make it green like the mac version ?

Reply:
missing word, `the ~/Android directory already`

Reply:
np